features/datum.feature

```diff
@@ -27,66 +124,136 @@ Functionaliteit: leveren van een datum
       | opschortingBijhouding | datum                                 | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
-      | verblijfplaats        | datumInschrijvingInGemeente           | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
-      | verblijfplaats        | datumVestigingInNederland             | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
-      | verblijfplaats        | datumIngangGeldigheid                 | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
       | verblijfstitel        | datumIngang                           | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
       | verblijfstitel        | datumEinde                            | JaarMaandDatum | 20300700 |            | 2030 | 7     |          | juli 2030    |
       | kiesrecht             | einddatumUitsluitingEuropeesKiesrecht | JaarMaandDatum | 20300700 |            | 2030 | 7     |          | juli 2030    |
       | kiesrecht             | einddatumUitsluitingKiesrecht         | JaarMaandDatum | 20300700 |            | 2030 | 7     |          | juli 2030    |
+      | immigratie            | datumVestigingInNederland             | Datum          | 20200308 | 2020-03-08 |      |       |          | 8 maart 2020 |
+      | immigratie            | datumVestigingInNederland             | DatumOnbekend  | 00000000 |            |      |       | true     | onbekend     |
+      | immigratie            | datumVestigingInNederland             | JaarDatum      | 20200000 |            | 2020 |       |          | 2020         |
+      | immigratie            | datumVestigingInNederland             | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
+
+    Abstract Scenario: <type> in <veld>
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde     |
+      | burgerservicenummer | 555550001  |
+      | <veld>              | <GbaDatum> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | <veld>                          |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam               | waarde        |
+      | <veld>.type        | <type>        |
+      | <veld>.datum       | <datum>       |
+      | <veld>.jaar        | <jaar>        |
+      | <veld>.maand       | <maand>       |
+      | <veld>.onbekend    | <onbekend>    |
+      | <veld>.langFormaat | <langFormaat> |
 
+      Voorbeelden:
+      | groep          | veld                        | type           | GbaDatum | datum      | jaar | maand | onbekend | langFormaat  |
```

Comment:
waarom de kolom groep? Die komt niet voor in de stappen. Bovendien zit dit veld niet meer in verblijfplaats (wel in de bron, niet in de API).

waarom kolom veld? Die is voor alle voorbeelden hetzelfde

Reply:
aangepast